server/src/employee/employee.module.ts

```diff
@@ -6,11 +6,12 @@ import { EmployeeService } from './employee.service';
 import { Manager } from 'src/manager/manager.entity';
 import { LocalStrategy } from 'src/authentication/local.strategy';
 import { AuthenticationService } from 'src/authentication/authentication.service';
+import { ManagerService } from 'src/manager/manager.service';
 
 @Module({
   imports: [TypeOrmModule.forFeature([Employee, Manager])],
```

Comment:
```suggestion
  imports: [TypeOrmModule.forFeature([Employee])],
```
vue que tu inject le `ManagerService` et plus le repos de la manager entity => plus besoin de l'entity Manager dans le module employee